### PR TITLE
fix: Configure the PortalNavigationIconUpgradePlugin to set icons to the portal navigation nodes - EXO-69848

### PR DESCRIPTION
**Pull request by sofyenne**
Prior to this change after migrating from the 6.4 to the 6.5 then to the 6.6 version , the external stream node navigation icon was incorrect , this change is going to configure the PortalNavigationIconUpgradePlugin in order to set icons for the portal navigation nodes .

### CODE REVIEW

**Review by azayati — CHANGES_REQUESTED** [review]
@sofyenne have you tested that on pgsql ?

**Comment by sofyenne** [comment]
> @sofyenne have you tested that on pgsql ?

Yep!

File: data-upgrade-navigations/src/test/java/org/exoplatform/migration/PortalNavigationIconMigrationTest.java [hunk 1]
```diff
@@ -0,0 +1,103 @@
+package org.exoplatform.migration;
```
Comment (azayati) [inline data-upgrade-navigations/src/test/java/org/exoplatform/migration/PortalNavigationIconMigrationTest.java:17]: put the header

**Review by azayati — CHANGES_REQUESTED** [review]

**Review by azayati — APPROVED** [review]
LGTM

**Review by Jihed525 — APPROVED** [review]

**Pull request merged**